content/docs/command-reference/plot/index.md

```diff
@@ -0,0 +1,188 @@
+# plot
+
+Contains commands to visualize users data stored in structured files like JSON,
+CSV, TSV: [show](/doc/command-reference/plot/show).
+[diff](/doc/command-reference/plot/diff).
+
+## Synopsis
+
+```usage
+usage: dvc plot [-h] [-q | -v] {show,diff} ...
+positional arguments:
+  {show,diff}    Use `dvc plot CMD --help` to display command-specific help.
+    show         Plot data from a file
+    diff         Plot changes between commits in the DVC repository,
+                 or between the last commit and the workspace.
+```
+
+## Description
+
+DVC provides a set of commands to visualize
+[continuous metrics](/doc/command-reference/plot#continous-metrics) of machine
+learning experiments in <abbr>DVC projects</abbr>. These metrics are usually
+plotted as AUC curves, loss functions, confusion matrixes and other types of
+plots.
+
+The continuous metrics should be saved in files which are usually created by
+users or generated by user's modeling or data processing code. The plot commands
+can work with these files commited to a repository history, data files
+controlled by DVC or files from workspace.
+
+### Continous metrics
+
+DVC has two concepts for metrics for representing result of machine learning
+training or data processing:
+
+1. `dvc metrics` to represent scalar numbers such as AUC, true positive rate and
+   others.
+2. `dvc plot` to visualize continuous metrics such as AUC curve, loss function,
+   confusion matrixes and others.
```

Comment:
Question: maybe we should use "scalar metrics" everywhere i.e. in the dvc metrics commands and other places where they're currently mentioned? Always try to use either scalar or continuous except in placs where metrics are mentioned in an abstract sense.

Reply:
> maybe we should use "scalar metrics" everywhere

Yes. It should be a very clear separation.

Reply:
OK added a checkbox to #1175